src/components/toggle-switch.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import { html, define } from 'hybrids';
+import { t } from '/vendor/@whotracksme/ui/src/i18n.js';
```

Comment:
cannot we import from `@whotracksme/ui/src/i18n.js`?

Reply:
Yes, but then we will have duplicated `i18n.js` file in the output, as the `serp-report/src/iframe/..` uses relative import (and it will take it from the vendor path).